charts/nginz/templates/conf/_nginx.conf.tpl

```diff
@@ -256,6 +254,12 @@ http {
         limit_conn conns_per_addr 20;
                 {{- end }}
               {{- end }}
+            {{- else }}
+              {{- if ($location.unlimited_requests_endpoint) }}
+                 # Note that this endpoint has no rate limit per user for autenticated requests
+              {{- else }}
+                 limit_req zone=reqs_per_user burst=20;
+              {{- end }}
```

Comment:
Few lines above there is also `limit_req zone=reqs_per_addr burst=10 nodelay;`. Do you think this will also cause issues?

Reply:
That is inside the "disable-zauth" block; so should not cause effects.

Overall, we don't have tests for rate limiting, unfortunately. Perhaps we should write some?